.travis.yml

```diff
@@ -14,6 +14,34 @@ matrix:
       script:
         - PYFLAKES_NODOCTEST=1 pyflakes scipy benchmarks/benchmarks | grep -E -v 'unable to detect undefined names|assigned to but never used|imported but unused|redefinition of unused' > test.out; cat test.out; test \! -s test.out
         - pep8 scipy benchmarks/benchmarks
+      after_success:
+        # run the doctests
+        - sudo apt-get update -qq
+        - sudo apt-get install -qq libatlas-dev libatlas-base-dev liblapack-dev gfortran
+        - sudo apt-get build-dep -qq python-matplotlib
+        - travis_retry pip install --install-option="--no-cython-compile" Cython==0.22
+        - travis_retry pip install $NUMPYSPEC
+        - travis_retry pip install nose argparse
+        - travis_retry pip install matplotlib
+        - python runtests.py --doctests-only -s linalg
```

Comment:
Agree, this is rather suboptimal :-). This is in fact the same limitation as with refguide_check which only deals with submodules. Can surely be fixed, but it'd entail messing up with either test discovery or hooking up the modified doctester into the nose machinery which gives me headache. So I opted for a quick-and-dirty solution here.